### DISCUSSION TITLE
Cannot build Deep-Atlas pipeline on my own dataset

### DISCUSSION THREAD

**Discussion by YuliangXiaoYLX** [Project-MONAI/MONAI#4457] [Q&A] [answered]
## Issue
I followed the deep_atlas tutorial to construct a pipeline for my own dataset. I downsample the dataset to 256x256x128 and the batch size for loading dataset for both registration and segmentation network are 4. However, I got an error below. Then, I tried to decrease the batch size to 1, but still got same errors, could anyone help me solve this issue? I am appreciated your help!

## Result
![Screen Shot 2022-05-31 at 22 27 17](https://user-images.githubusercontent.com/70731556/171316053-719e5639-932a-4f9b-9535-eec07fff62cd.png)

## System Info
![Screen Shot 2022-05-31 at 22 38 39](https://user-images.githubusercontent.com/70731556/171316780-fa338c20-a3b0-49ff-a76f-55b71514165c.png)

**Comment by YuliangXiaoYLX** [comment, marked as answer]
This problem was already solved, please see following explanation!!!

**Reply by wyli** [reply]
thanks for getting back, would you be interested in sharing your solution here? it's potentially useful for other users who arrive here through search engines.

**Reply by ebrahimebrahim** [reply]
It looks like a CUDA issue of the sort I sometimes mysteriously resolve by rebooting

**Reply by YuliangXiaoYLX** [reply]
@wyli Sure, it is the problem of the version of Torch. Since my CUDA version is 11.4. But I used the MONAI default installation which installed the Torch 1.11.0+cu102. As a result, I uninstalled the old version and re-installed Torch 1.11.0+cu113 which is compatible to my GPU CUDA.

**Reply by YuliangXiaoYLX** [reply]
> It looks like a CUDA issue of the sort I sometimes mysteriously resolve by rebooting
it is the problem of the version of Torch. Since my CUDA version is 11.4. But I used the MONAI default installation which installed the Torch 1.11.0+cu102. As a result, I uninstalled the old version and re-installed Torch 1.11.0+cu113 which is compatible to my GPU CUDA.
